docs/modules/troubleshooting/pages/startup-issues.adoc

```diff
@@ -0,0 +1,121 @@
+
+= {page-component-title} Startup
+
+This section describes troubleshooting steps for problems encountered while {page-component-title} starts.
+
+[[java-errors]]
+== Java errors
+
+If you encounter a Java error when starting {page-component-title}, you should verify that your Java version is compatible:
+
+[source, console]
+runjava -s
+
+If you run `systemctl start opennms` with a version of Java that is not compatible, the output displays error messages:
+
+[source, shell]
+----
+opennms[80542]: Starting OpenNMS:
+opennms[81178]: runjava: Error: configured JVM not found.
+opennms[81178]: runjava: "/usr/lib/jvm/java-11-openjdk-11.0.17.0.8-1.el9_1.x86_64/bin/java" does not exist or is not executab>
+opennms[81178]: runjava: Run "/opt/opennms/bin/runjava -s" to set up the java.conf file.
+opennms[81178]: runjava:
+opennms[81178]: runjava: You can install a JVM by downloading one from Oracle, or by running something like
+opennms[81178]: runjava: 'apt-get install openjdk-11-jdk' or 'yum install java-11-openjdk-devel'.
+opennms[80542]: OpenNMS was unable to connect to the 'opennms' database configured in opennms-datasources.xml.
+opennms[80542]: failed
+systemd[1]: opennms.service: Control process exited, code=exited, status=1/FAILURE
+----
+
+See xref:deployment:core/system-requirements.adoc[] for software compatibility requirements, including the supported versions of Java.
+
+[[startup-errors]]
+== Configuration errors
+
+If you encounter a configuration error when starting {page-component-title}, you should ensure that {page-component-title} is installed and updated to the latest version:
+
+[source, console]
+install -dis
+
+If you run `systemctl start opennms` and {page-component-title} is not installed or up to date, the output displays error messages:
```

Comment:
Please use a variable in this command spec, so that it renders as `systemctl stop meridian` when the docs are built as Meridian. I'm in a hurry so apologies for not looking up that variable name.

Reply:
The systemd unit is never `meridian`.  it is always `opennms`.